packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -979,10 +979,17 @@ async function handler(
 
   const allCredentials = await getAllCredentials(organizerUser, eventType);
 
-  const isOrganizerRescheduling = organizerUser.id === userId;
+  const { userReschedulingIsOwner, isConfirmedByDefault } = getRequiresConfirmationFlags({
+    eventType,
+    bookingStartTime: reqBody.start,
+    userId,
+    originalRescheduledBookingOrganizerId: originalRescheduledBooking?.user?.id,
+    paymentAppData,
+  });
 
+  // If the Organizer himself is rescheduling, the booker should be sent the communication in his timezone and locale.
   const attendeeInfoOnReschedule =
-    isOrganizerRescheduling && originalRescheduledBooking
+    userReschedulingIsOwner && originalRescheduledBooking
```

Comment:
Removed `isOrganizerRescheduling` in favour of `userReschedulingIsOwner` which is a better way to detect if the booking organizer is rescheduling